src/main/kotlin/app/revanced/patches/youtube/layout/hide/rollingnumber/fingerprints/RollingNumberTextViewAnimationUpdateFingerprint.kt

```diff
@@ -0,0 +1,30 @@
+package app.revanced.patches.youtube.layout.hide.rollingnumber.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+import com.android.tools.smali.dexlib2.Opcode
+
+object RollingNumberTextViewAnimationUpdateFingerprint : MethodFingerprint(
+    returnType = "V",
+    accessFlags = AccessFlags.PUBLIC or AccessFlags.FINAL,
+    parameters = listOf("Landroid/graphics/Bitmap;"),
+    opcodes = listOf(
+        Opcode.NEW_INSTANCE, // bitmap ImageSpan
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT_OBJECT,
+        Opcode.CONST_4,
+        Opcode.INVOKE_DIRECT,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT,
+        Opcode.CONST_16,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT,
+        Opcode.INT_TO_FLOAT,
+        Opcode.INVOKE_VIRTUAL, // set textview padding using bitmap width
+    ),
+    customFingerprint = { _, classDef ->
+        classDef.superclass == "Landroid/support/v7/widget/AppCompatTextView;"
+    }
+)
```

Comment:
I think this fingerprint is similar to another one and both can be moved under the shared package so that they can be used in both patches where they are needed. In relation to my other comment, the third patch only could need it.